Content.Server/Chemistry/EntitySystems/ChemMasterSystem.cs

```diff
@@ -93,6 +97,19 @@ private void OnSetModeMessage(Entity<ChemMasterComponent> chemMaster, ref ChemMa
             ClickSound(chemMaster);
         }
 
+        // Begin DeltaV - chemmaster sources
+        private void OnSetSourceMessage(Entity<ChemMasterComponent> chemMaster, ref ChemMasterSetSourceMessage message)
+        {
+            // Ensure the mode is valid, either Transfer or Discard.
+            if (!Enum.IsDefined(typeof(ChemMasterSource), message.ChemMasterSource))
```

Comment:
i imagine when deserializing enums this is already done since nothing else in the game checks this